src/appkit.cs

```diff
@@ -744,6 +746,10 @@ public interface NSApplication : NSAccessibilityElementProtocol, NSAccessibility
 		[Export ("orderFrontStandardAboutPanelWithOptions:")]
 		void OrderFrontStandardAboutPanelWithOptions (NSDictionary optionsDictionary);
 #endif
+
+		[Mac (10,12)]
+		[Export ("enumerateWindowsWithOptions:usingBlock:")]
+		unsafe void EnumerateWindows (NSWindowListOptions options, Action<NSWindow, /*bool* */ IntPtr> block);
```

Comment:
* `block` should be named something like `handler` or `completion` depending on what the documentation says if public
* `unsafe` keyword is not needed here
* `Action<NSWindow, /*bool* */ IntPtr> block` this should have its own delegate signature because the user  won't know what to do with the second argument of the callback. I think you can use something like `delegate void NSApplicationEnumerateWindowsHandler (NSWindow window, ref bool stop);`. From what I can get from [docs](https://developer.apple.com/reference/appkit/nsapplication/1644472-enumeratewindowswithoptions?language=objc) the user will need to set the `stop` to `true` in order to stop the enumeration.